lib/shared/schema-validation-error.js

```diff
@@ -0,0 +1,11 @@
+"use strict";
+
+/** Exception thrown when schema validation fails because a rule is provided invalid options. */
```

Comment:
Matching other formatting

```suggestion
/**
 * Exception thrown when schema validation fails because a rule is provided invalid options.
 */
```